controller-api/src/main/proto/responsive/controller/v1/controller.proto

```diff
@@ -73,32 +73,34 @@ enum PolicyStatus {
 
 // TODO(rohan): move this to a a common location where we specify the policy
 //              also, see if we can combine with the CRD somehow
-message ApplicationPolicy {
+message ApplicationPolicySpec {
```

Comment:
can we encode a version somewhere in the spec? I'd feel better if we could throw exceptions if the versions mismatch

Reply:
actually, I think it makes more sense to have it at the top level (wherever that schema is) - let me know if we already have one.

Reply:
yeah the proto file itself is versioned. So if we want to change the protocol we can just add a new set of protos under `controller-api/src/main/proto/responsive/controller/v2` and name the service something like `ControllerV2`